internal/service/events/rule_data_source_test.go

```diff
@@ -0,0 +1,62 @@
+package events_test
+
+import (
+	"fmt"
+	"testing"
+
+	sdkacctest "github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+)
+
+func TestAccEventsRuleDataSource_basic(t *testing.T) {
+
+	ruleName := fmt.Sprintf("test_rule-%d", sdkacctest.RandInt())
+	busName := fmt.Sprintf("test_bus-%d", sdkacctest.RandInt())
```

Comment:
The common practice is to use `sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)`.

Please have a look at other acceptance tests as a reference. E.g.:
https://github.com/hashicorp/terraform-provider-aws/blob/d1e68d3f2987a8e75401d39df7b226ebb3d7c5b2/internal/service/identitystore/user_data_source_test.go#L15-L18